spec/features/bot_with_right_avatar_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+feature "user sees the right avatar for his bot", %{
+  As an authenticated user
+  I want to see an avatar of my bot that fits his description
+  So that I can my bot can have the right avatar
+} do
+
+
+  scenario "user sees the right avatar for a male,blonde, green eyes bot " do
+    user = FactoryGirl.create(:user)
+    mary = FactoryGirl.create(
+      :bot,
+      gender: 'Male',
+      hair_color: 'Blonde',
+      eye_color: 'Green',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.